src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1065,6 +1065,31 @@ def test_functionapp_consumption_e2e(self, resource_group, storage_account):
 
         self.cmd('functionapp delete -g {} -n {}'.format(resource_group, functionapp_name))
 
+    @ResourceGroupPreparer(name_prefix='azurecli-functionapp-c-e2e-ragrs', location='westus')
+    @StorageAccountPreparer(sku='Standard_RAGRS')
+    def test_functionapp_e2e_ragrs_storage(self, resource_group, storage_account):
+        functionapp_name = self.create_random_name('functionappconsumption', 40)
+
+        self.cmd('functionapp create -g {} -n {} -c westus -s {}'
+                 .format(resource_group, functionapp_name, storage_account)).assert_with_checks([
+                     JMESPathCheck('state', 'Running'),
+                     JMESPathCheck('name', functionapp_name),
+                     JMESPathCheck('hostNames[0]', functionapp_name + '.azurewebsites.net')])
+
+        self.cmd('functionapp list -g {}'.format(resource_group), checks=[
+            JMESPathCheck('[0].kind', 'functionapp'),
+            JMESPathCheck('[0].name', functionapp_name)
+        ])
+        self.cmd('functionapp show -g {} -n {}'.format(resource_group, functionapp_name), checks=[
+            JMESPathCheck('kind', 'functionapp'),
+            JMESPathCheck('name', functionapp_name)
+        ])
+        self.cmd('functionapp update -g {} -n {} --set clientAffinityEnabled=true'.format(resource_group, functionapp_name), checks=[
+            self.check('clientAffinityEnabled', True)
+        ])
+
+        self.cmd('functionapp delete -g {} -n {}'.format(resource_group, functionapp_name))
```

Comment:
I think we do not need to explicitly do `delete` because it the resource-group gets deleted later and takes care of it. I was told that deleting like this would slow the tests.

Reply:
It is true, unless the test is purposefully testing the delete command.